lib/StripeResource.js

```diff
@@ -125,6 +125,8 @@ StripeResource.prototype = {
         // lastResponse.
         res.requestId = headers['request-id'];
 
+        var requestDuration = Date.now() - req._requestStart;
```

Comment:
could you make this variable include the units in the name?  (e.g. suffix it with "ms")?